### PR TITLE
extended timeout for frontend and commiting mixed loss for dl training

### PR DESCRIPTION
**Pull request by notthattal**
- Extended the timeout on the frontend to display fallback recommendations from 5 seconds to 30 seconds
- Updated the requirements.txt to reflect the necessary imports
- Updated the train_model function in the deep learning model to use both BPR and MSE loss functions with a tunable hyperparameter alpha that sets the total loss function to total_loss = alpha*mse_loss + (1-alpha)*bpr_loss

### CODE REVIEW

**Comment by ahmed-boutar** [comment]
Looks good to me! Please update the deep learning section to talk about the new changes in the loss
